eng/ApiCompat/ApiCompat.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="VerifyCompat">
+
```

Comment:
Can we add a comment up here that briefly summarizes what's happening and points out all the moving pieces?  I think partner teams may end up in this file at some point while trying to understand things and it'd be great to give them a bit of help.

Reply:
Added